.vscode/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Is this json works in .vscode? Technically comments are not allowed in JSON

Reply:
yes. it works.
The base file with this comments was auto generated. I  added more configuration